setup.cfg

```diff
@@ -41,6 +41,9 @@ datalad.extensions =
     # valid datalad interface specification (see demo in this extensions)
     container = datalad_container:command_suite
 
+datalad.metadata.extractors =
+    metalad_container = datalad_container.extractors.metalad_container:MetaladContainer
```

Comment:
Easier than expected because this endpoint is under `datalad.metalad`

Without `metalad` installed:

``` 
datalad meta-extract -d . singularity_inspect images/bids/bids-pymvpa--1.0.2.sing  | jq
datalad: Unknown command 'meta-extract'.  See 'datalad --help'.

Did you mean any of these?
        metadata
```

With `metalad` installed.

```
{
  "type": "file",
  "dataset_id": "b02e63c2-62c1-11e9-82b0-52540040489c",
  "dataset_version": "9ed0a39406e518f0309bb665a99b64dec719fb08",
  "path": "images/bids/bids-pymvpa--1.0.2.sing",
  "extractor_name": "singularity_inspect",
  "extractor_version": "0.0.1",
  "extraction_parameter": {},
  "extraction_time": 1678382512.413365,
  "agent_name": "Austin Macdonald",
  "agent_email": "austin@dartmouth.edu",
  "extracted_metadata": {
    "@id": "datalad:SHA1-s993116191--cc7ac6e6a31e9ac131035a88f699dfcca785b844",
    "type": "file",
    "path": "images/bids/bids-pymvpa--1.0.2.sing",
    "content_byte_size": 0,
    "comment": "SingularityInspect extractor executed at 1678382512.2972443",
    "singularity_version": "apptainer version 1.1.5-1.fc37",
    "singularity_inspect": {
      "data": {
        "attributes": {
          "labels": {
            "org.label-schema.build-date": "Thu,_19_Dec_2019_14:58:41_+0000",
            "org.label-schema.build-size": "2442MB",
            "org.label-schema.schema-version": "1.0",
            "org.label-schema.usage.singularity.deffile": "Singularity.bids-pymvpa--1.0.2",
            "org.label-schema.usage.singularity.deffile.bootstrap": "docker",
            "org.label-schema.usage.singularity.deffile.from": "bids/pymvpa:v1.0.2",
            "org.label-schema.usage.singularity.version": "2.5.2-feature-squashbuild-secbuild-2.5.6e68f9725"
          }
        }
      },
      "type": "container"
    }
  }
}
```

Reply:
coolio then